pkg/ride/tree_evaluation_test.go

```diff
@@ -10227,178 +10227,3 @@ func TestInvokeDappAttachedPaymentsLimitAfterV6(t *testing.T) {
 	require.NoError(t, err)
 	require.NotNil(t, res)
 }
-
-func TestZeroPaymentsV6(t *testing.T) {
```

Comment:
May be better not to remove the test, but remove the last check? If there is no test duplication.

Reply:
Yep, changed name to more appropriate and left checks for both cases rideV6Activated=false/true.